src/frontend/js/utils/test/factories/richie.ts

```diff
@@ -0,0 +1,62 @@
+import { compose, createSpec, derived, faker } from '@helpscout/helix';
+import { APIBackend } from 'types/api';
+import { CommonDataProps } from 'types/commonDataProps';
+
+export const CourseStateFactory = createSpec({
+  priority: derived(() => Math.floor(Math.random() * 7)),
+  datetime: derived(() => faker.date.past()().toISOString()),
+  call_to_action: faker.random.words(1, 3),
+  text: faker.random.words(1, 3),
+});
+
+export const CourseRunFactory = createSpec({
+  id: faker.datatype.number(),
+  resource_link: faker.unique(faker.internet.url()),
+  start: derived(() => faker.date.past()().toISOString()),
+  end: derived(() => faker.date.past()().toISOString()),
+  enrollment_start: derived(() => faker.date.past()().toISOString()),
+  enrollment_end: derived(() => faker.date.past()().toISOString()),
+  languages: faker.random.locale(),
+  state: CourseStateFactory,
+  starts_in_message: null,
+});
+
+export const EnrollmentFactory = createSpec({
+  id: faker.datatype.number(),
+  created_at: derived(() => faker.date.past()().toISOString()),
+  user: faker.datatype.number(),
+  course_run: faker.datatype.number(),
+});
+
+export const UserFactory = createSpec({
+  full_name: faker.fake('{{name.firstName}} {{name.lastName}}'),
+  username: faker.internet.userName(),
+});
+
+export const FonzieUserFactory = compose(
+  UserFactory,
+  createSpec({
+    access_token: btoa(faker.datatype.uuid()),
+  }),
+);
+
+export const RichieContextFactory = (context: Partial<CommonDataProps['context']> = {}) =>
```

Comment:
Why not conserve `ContextFactory` and prevent too much changes ? Furthermore, there is no duplication issue about this factory.
```suggestion
export const ContextFactory = (context: Partial<CommonDataProps['context']> = {}) =>
```

Reply:
`Context` is kind of a reserved word in React.
here we talk about the richie's application context and not a ReactContextFactory.

Reply:
Mokay